content/docs/setup/kubernetes/install/cni/index.md

```diff
@@ -45,30 +45,73 @@ replaces the functionality provided by the `istio-init` container.
     1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
         settings required.
 
-1.  Add the `--set istio_cni.enabled=true` setting to the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/) to include the installation of the Istio CNI plugin. For example:
+1.  Update Helm's local package cache per the instructions in the [Istio Installation with Helm procedure](/docs/setup/kubernetes/install/helm/).
+
+    1.  If installing via `helm template` fetch the `istio-cni` template in addition to the other Istio
+        templates fetched per the [Istio Helm installation procedure](/docs/setup/kubernetes/install/helm/#option-1-install-with-helm-via-helm-template)
+
+        {{< text bash >}}
```

Comment:
I think you need some kind of note here to indicate the CNI charts are pushed to the same locations that the istio charts are pushed to.   As we discussed directly I don't think you want to get in the version quagmire of which specific location to grab from.   For most people it may not be obvious that the CNI charts are pushed to the same locations.

Reply:
IMHO, since this is user facing doc they shouldn't care about those release & src repo details.  Doing the `helm repo add istio.io ...` instructions always makes these CNI instructions work.  So, I'm wary to add more complicated info here.

Reply:
@john-a-joyce We can add the details of the implementation in a separate document at a later date. Adding the details here could confuse and over-complicate the procedure.